data/core/units/orcs/Shaman.cfg

```diff
@@ -0,0 +1,99 @@
+#textdomain wesnoth-units
+
+[unit_type]
+    id=Orcish Shaman
```

Comment:
ID might possibly clash with several UMC.

Reply:
I'm not really sure what I'm supposed to do about that…